pkg/controllers/raycluster_controller.go

```diff
@@ -220,7 +222,7 @@ func crbNameFromCluster(cluster *rayv1.RayCluster) string {
 func desiredOAuthClusterRoleBinding(cluster *rayv1.RayCluster) *rbacapply.ClusterRoleBindingApplyConfiguration {
 	return rbacapply.ClusterRoleBinding(
```

Comment:
I really wonder, can this be a RoleBinding instead of a ClusterRoleBinding?

Reply:
I believe I tried this a while back and it didn't work. I can give it another go though